src/main/java/ca/ciccc/Assignment1.java

```diff
@@ -127,7 +171,24 @@ public static String powers(int number) {
      *          Min integer: 5"
      */
     public static String arithmetic(int a, int b) {
-
-        return "";
+        int sum = a+b;
+        int diff = a-b;
+        int prod = a*b;
+        double average = (a+b)/2;
+        int dist;
+        if ((a<0 && b>0) || (a>0 && b<0)){
```

Comment:
(a<0 && b>0) is OK, but in this case (a>0 && b<0), answer is different. It's gonna be minus.

Reply:
You can also use `Math.abs(a)` for absolute value